rails-commenting.rb

```diff
@@ -34,13 +38,15 @@ def create
   end
 
   # ---6)
+  #  below is the method that contains a param that allows for a saved blog post to be edited. The params accepts the id of the post and the proper info is found.
```

Comment:
The edit page will return a HTML form that allows the creation for a new blog post.